src/axom/sidre/core/MFEMSidreDataCollection.hpp

```diff
@@ -7,13 +7,14 @@
 #define MFEM_SIDRE_DATACOLLECTION_HPP_
 
 #include "axom/config.hpp"
+#include "DataStore.hpp"
+#include "Group.hpp"
+#include "View.hpp"
```

Comment:
Including these three headers in favor of sidre.hpp is just a refactoring, right?  I like this; why did you make this change?

Reply:
(please see my response to a similar comment on `MFEMSidreDataCollection`)